craft_parts/overlays/chroot.py

```diff
@@ -104,8 +106,8 @@ def _cleanup_chroot(path: Path) -> None:
     _Mount(None, "/etc/resolv.conf", "/etc/resolv.conf", "--bind"),
     _Mount("proc", "proc", "/proc", None),
     _Mount("sysfs", "sysfs", "/sys", None),
-    _Mount(None, "/dev", "/dev", "--bind"),
-    _Mount("tmpfs", "tmpfs", "/dev/shm", None),
```

Comment:
how come the `tmpfs` mount was removed?

Reply:
oh is it because its mounted under `/dev`?

Reply:
Yes, it's mounted under /dev by `--rbind`, and removed with `--recursive`.